src/Lookup/Lookup.tsx

```diff
@@ -419,6 +419,10 @@ export default class Lookup extends React.PureComponent<LookupProps, State> {
 	};
 
 	onMenuOpen = async () => {
+		if (this.state.menuIsOpen)  {
```

Comment:
What is the reason for this to be triggered on an already opened menu?

Reply:
Not sure, to be honest. I guess it is the `Select` component that triggers its onMenuOpen function when you type into the textfield. And the Select componet is a thirdparty lib,  so i didn't find a better solution than checking if it's opened within the function.

Reply:
Known issue. Please leave a comment to followup this PR https://github.com/JedWatson/react-select/pull/3897